.rubocop.yml

```diff
@@ -0,0 +1,12 @@
+# See https://docs.rubocop.org/rubocop/configuration
+require:
+  - rubocop-rspec
+  - rubocop-rails
+
+AllCops:
+  NewCops: enable
+
+RSpec/FactoryBot/CreateList:
+  Include:
+    - 'spec/factories.rb'
+    - 'spec/factories/**/*.rb'
```

Comment:
This shouldn't be necessary as it's the default option for this Cop, but for some reason Rubocop was trying to run the cop on the wrong files.